src/main/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperation.java

```diff
@@ -1,10 +1,11 @@
 package uk.gov.hmcts.ccd.domain.service.startevent;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import com.google.common.collect.Maps;
```

Comment:
mmm isn't it a bit too late to set the default values in the operation classes? For example, when invoking a MidEvent callback CCD sends to the midevent callback the case data, here: 

https://github.com/hmcts/ccd-data-store-api/blob/80adc76067063ba3c3600fb3e0674b41bfe5426f/src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/MidEventCallback.java#L69

With this implementation the callback is not receiving the default value right?

Reply:
@mario-paniccia 
Do we need to worry about the mid event callback, as we default values will be populated in the about to start event callback already....

Reply:
This thing is that the MidEven also sends the caseDetailsBefore fields, which should contain the case data before the modifications done in the event. Now, should that case data contain default values? In my opinion yes. If we say this fields is defaulted to a value, then in CCD that field has got that initial value.

Reply:
@rafalkalita let's ask Nigel please. It would also be ok for now to ignore this comment.

Reply:
Nigel confirmed we do not need this: "Defaults only work for start event so not for midevent"